### PR TITLE
 Add files needed to push to the aur when 1.0 

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**

### CODE REVIEW

File: .SRCINFO [hunk 1]
```diff
@@ -0,0 +1,13 @@
+pkgbase = autoclockspeed-bin
+	pkgdesc = A utility to check stats about your CPU, and auto regulate clock speeds to help with either performance or battery life.
+	pkgver = 0.1.9
```
Comment (Camerooooon) [inline .SRCINFO:3]: Don't forget to change to 1.0.0

File: PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#
+# This PKGBUILD was generated by `cargo aur`: https://crates.io/crates/cargo-aur
+
+pkgname=autoclockspeed-bin
+pkgver=0.1.9
+pkgrel=1
+pkgdesc="A utility to check stats about your CPU, and auto regulate clock speeds to help with either performance or battery life."
+url="https://github.com/JakeRoggenbuck/auto-clock-speed"
+license=("MIT")
+arch=("x86_64")
```
Comment (Shuzhengz) [inline PKGBUILD:13]: Are we not going to support ARM then?

File: .SRCINFO [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+	pkgrel = 1
+	url = https://github.com/JakeRoggenbuck/auto-clock-speed
+	arch = x86_64
```
Comment (Shuzhengz) [inline .SRCINFO:6]: same here

**Review by Shuzhengz — APPROVED** [review]
lgtm but a small question

File: PKGBUILD (hunk 2 above)
Comment (JakeRoggenbuck) [reply PKGBUILD:13]: Hmm, should we? I'd be open to it

Comment (Shuzhengz) [reply PKGBUILD:13]: We could, but I don't have any device to test it with so yea theres that

Comment (JakeRoggenbuck) [reply PKGBUILD:13]: would a raspberry pi work?

Comment (Shuzhengz) [reply PKGBUILD:13]: i think so, but we prob want more variety

Comment (JakeRoggenbuck) [reply PKGBUILD:13]: yea

**Pull request merged**